BuildServer/Windows/Resources/nsis/MDANSE_installer.nsi

```diff
@@ -20,18 +20,18 @@ OutFile "${TARGET_DIR}\MDANSE-${VERSION}-${ARCH}.exe"
 RequestExecutionLevel admin #NOTE: You still need to check user rights with UserInfo!
 
 ; The default installation directory
-InstallDir "$PROGRAMFILES\Institut Laue-Langevin\MDANSE"
+InstallDir "$PROGRAMFILES\MDANSE"
 
 ; Registry key to check for directory (so if you install again, it will overwrite the old one automatically)
-InstallDirRegKey HKLM "Software\Institut Laue-Langevin\MDANSE" "Install_Dir"
+InstallDirRegKey HKLM "Software\MDANSE" "Install_Dir"
 
 ; Will show the details of installation
 ShowInstDetails show
 
 ; Will show the details of uninstallation
 ShowUnInstDetails show
 
-!define PUBLISHER "Institut Laue-Langevin"
+!define PUBLISHER "ILL"
```

Comment:
Should we change this to ISIS Neutron and Muon Source? Or something else?

Reply:
Yes, please change this to ISIS Neutron and Muon Source